dkg/runner.go

```diff
@@ -69,78 +76,79 @@ func (r *Runner) ProcessMsg(msg *SignedMessage) (bool, map[types.OperatorID]*Sig
 				Signature: sig.Serialize(),
 			}
 			if err := r.signAndBroadcastMsg(pdd, DepositDataMsgType); err != nil {
-				return false, nil, errors.Wrap(err, "could not broadcast partial deposit data")
+				return false, errors.Wrap(err, "could not broadcast partial deposit data")
 			}
 			r.DepositDataSignatures[r.Operator.OperatorID] = pdd
 		}
-		return false, nil, nil
+		return false, nil
 	case DepositDataMsgType:
 		depSig := &PartialDepositData{}
 		if err := depSig.Decode(msg.Message.Data); err != nil {
-			return false, nil, errors.Wrap(err, "could not decode PartialDepositData")
+			return false, errors.Wrap(err, "could not decode PartialDepositData")
 		}
 
 		if err := r.validateDepositDataSig(depSig); err != nil {
-			return false, nil, errors.Wrap(err, "PartialDepositData invalid")
+			return false, errors.Wrap(err, "PartialDepositData invalid")
 		}
 
 		if found := r.DepositDataSignatures[msg.Signer]; found == nil {
 			r.DepositDataSignatures[msg.Signer] = depSig
 		} else if !bytes.Equal(found.Signature, msg.Signature) {
-			return false, nil, errors.New("inconsistent partial signature received")
+			return false, errors.New("inconsistent partial signature received")
 		}
 
 		if len(r.DepositDataSignatures) == int(r.InitMsg.Threshold) {
 			// reconstruct deposit data sig
 			depositSig, err := r.reconstructDepositDataSignature()
 			if err != nil {
-				return false, nil, errors.Wrap(err, "could not reconstruct deposit data sig")
+				return false, errors.Wrap(err, "could not reconstruct deposit data sig")
 			}
 
 			// encrypt Operator's share
-			encryptedShare, err := r.config.Signer.Encrypt(r.Operator.EncryptionPubKey, r.KeyGenOutput.Share.Serialize())
+			encryptedShare, err := r.config.Signer.Encrypt(r.Operator.EncryptionPubKey, r.KeygenOutcome.KeyGenOutput.Share.Serialize())
 			if err != nil {
-				return false, nil, errors.Wrap(err, "could not encrypt share")
+				return false, errors.Wrap(err, "could not encrypt share")
 			}
 
 			ret, err := r.generateSignedOutput(&Output{
 				RequestID:            r.Identifier,
 				EncryptedShare:       encryptedShare,
-				SharePubKey:          r.KeyGenOutput.Share.GetPublicKey().Serialize(),
-				ValidatorPubKey:      r.KeyGenOutput.ValidatorPK,
+				SharePubKey:          r.KeygenOutcome.KeyGenOutput.Share.GetPublicKey().Serialize(),
+				ValidatorPubKey:      r.KeygenOutcome.KeyGenOutput.ValidatorPK,
 				DepositDataSignature: depositSig,
 			})
 			if err != nil {
-				return false, nil, errors.Wrap(err, "could not generate dkg SignedOutput")
+				return false, errors.Wrap(err, "could not generate dkg SignedOutput")
 			}
 
 			r.OutputMsgs[r.Operator.OperatorID] = ret
 			if err := r.signAndBroadcastMsg(ret, OutputMsgType); err != nil {
-				return false, nil, errors.Wrap(err, "could not broadcast SignedOutput")
+				return false, errors.Wrap(err, "could not broadcast SignedOutput")
 			}
-			return false, nil, nil
+			return false, nil
```

Comment:
can simply return errors.wrap(err, "...")

Reply:
Done in https://github.com/bloxapp/ssv-spec/pull/81/commits/8ebc91d835ba63c395826b6897f99d8105e76154